src/ledger/InMemoryLedgerTxn.h

```diff
@@ -50,6 +50,30 @@ class InMemoryLedgerTxn : public LedgerTxn
     void updateLedgerKeyMap(InternalLedgerKey const& genKey, bool add) noexcept;
     void updateLedgerKeyMap(EntryIterator iter);
 
+    class FilteredEntryIteratorImpl : public EntryIterator::AbstractImpl
+    {
+        EntryIterator mIter;
+
+      public:
+        FilteredEntryIteratorImpl(EntryIterator const& begin);
```

Comment:
`explicit FilteredEntryIteratorImpl`, see https://stackoverflow.com/questions/121162/what-does-the-explicit-keyword-mean.

Reply:
Done